product_docs/docs/pgd/5/quickstart/further_explore_replication.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Exploring replication with PGD
+navTitle: Exploring replication
+deepToC: true
+---
+
+## Explore replication with PGD
+
+With the cluster up and running, it's worthwhile to run some basic checks to see how effectively it's replicating.
+
+The following example shows one quick way to do this, but you must ensure that any testing you perform is appropriate for your use case.
+
+* **Preparation**
+  * Ensure the cluster is ready:  
+    * Log in to the database on first host. 
+    * Run `select bdr.wait_slot_confirm_lsn(NULL, NULL);`.  
+    * When the query returns, the cluster is ready.  
+
+* **Create data**
+  The simplest way to test that the cluster is replicating is to log in to a node, create a table, and populate it. On the first node, 
```

Comment:
```suggestion
  The simplest way to test that the cluster is replicating is to log in to a node, create a table, populate it, and see the data you populated appear on a second node. On the first node, 
```

Reply:
(Since creating a table and populating doesn't actually on its own tell you anything about replication)